src/main/resources/anniv6Resources/localization/eng/BrokenSpace/Relicstrings.json

```
@@ -0,0 +1,104 @@
+{
+  "${ModID}:BrokenDataDisk": {
+    "NAME": "Broken Data Disk",
+    "FLAVOR": "no",
+    "DESCRIPTIONS": [
+      "At the start of each combat, lose #b",
+      " #yFocus. The first time you channel an orb each turn, channel another orb of the same type."
```

I'm taking a look at minor stuff like wording consistency. Channel is a keyword and the base game consistently capitalizes Orb.
```suggestion
      " #yFocus. The first time you #yChannel an Orb each turn, #yChannel another Orb of the same type."
```